pkg/webui/console/components/webhook-form/index.js

```diff
@@ -370,6 +372,11 @@ export default class WebhookForm extends Component {
     this.setState({ shouldShowCredentialsInput: event.target.checked })
   }
 
+  @bind
+  handleHeadersChange() {
+    this.setState({ showDecodeError: !hasNoEmptyEntry })
+  }
+
```

Comment:
Where is this used?

Reply:
It is used [here](https://github.com/TheThingsNetwork/lorawan-stack/blob/a4897719221f849acccaa5f654e156e4faee7e24/pkg/webui/console/components/webhook-form/index.js#L541)